third_party/move/move-compiler-v2/tests/live-var/mut_ref.exp

```diff
@@ -162,14 +164,6 @@ fun m::some2($t0: &mut m::R, $t1: &mut m::R) {
   0: return ()
 }
 
-
```

Comment:
I think it is solved here: https://github.com/aptos-labs/aptos-core/pull/11213?

Reply:
you are right, it is strange. Let me take a look

Reply:
This appears after rebasing. Maybe it is a bug in the live analysis because it cannot detect the error for the following bytecode:
```
fun m::f2_fail() {
     var $t0: m::R
     var $t1: m::R
     var $t2: u64
     var $t3: &mut m::R
     var $t4: &mut m::R
     var $t5: &mut m::R
  0: $t2 := 0
  1: $t1 := pack m::R($t2)
  2: $t0 := infer($t1)
  3: $t4 := borrow_local($t0)
  4: $t3 := infer($t4)
  5: $t5 := infer($t3)
  6: m::some2($t5, $t3) // $t5 and $t3 are both &mut to $t1
  7: return ()
}
```

Reply:
Should the bytecode program above be rejected in the reference_safety_check? cc @wrwg

Reply:
This should really be caught by reference_safety, yes.  @rahxephon89 try adding ReferenceSafetyProcessor to the pipeline for "/live-var/" in testsuite.rs.  If it doesn't work, then @wrwg has more work to do, maybe.

Reply:
Maybe could be solved by the ability check: https://github.com/aptos-labs/aptos-core/issues/10723?

Reply:
This is expected and was already submitted, seems some result of rebase issue that it does show up here. This error is not longer handled in livevar but in reference analysis, which has its own tests about this scenario.

Reply:
Should I remove this test case then?

Reply:
Ah no, wait. Yes, this particular error is not longer generated in all situations because of the introduced temps, so just update the .exp is the way to go.